.travis.yml

```diff
@@ -26,8 +36,18 @@ addons:
     - pkg-config
     - clang-format-5.0
```

Comment:
we don't really need this package. only what's in here https://github.com/kinecosystem/blockchain-ops/blob/40ccfc49a18693f09d054c3ece5c194a01f67063/images/dockerfiles/Dockerfile.stellar-core-build#L4